tests/dummy/app/controllers/demo/select.js

```diff
@@ -7,7 +7,17 @@ export default Ember.Controller.extend({
     return 'AL AK AZ AR CA CO CT DE FL GA HI ID IL IN IA KS KY LA ME MD MA MI MN MS MO MT NE NV NH NJ NM NY NC ND OH OK OR PA RI SC SD TN TX UT VT VA WA WV WI WY'
             .split(' ').map((state) => ({ abbrev: state }));
   }),
-
+  vegetables: Ember.A([
+    {name: "Corn", checked: false},
+    {name: "Onions", checked: false},
+    {name: "Kale", checked: false},
+    {name: "Arugula", checked: false},
+    {name: "Peas", checked: false},
+    {name: "Zucchini", checked: false}]),
+  searchTerm: "",
+  filteredVegetables: Ember.computed.filter('vegetables',function(vegetable) {
+    return vegetable.name.toLowerCase().indexOf(this.get('searchTerm').toLowerCase()) > -1;
+  }).property('vegetables','searchTerm'),
```

Comment:
@miguelcobain Yes. It allows you to trigger the filter based on multiple properties. The only usage rule for this format is that in the .property(...) section, the first property MUST be the property that the .filter('property') is filtering.